htdocs/public/members/new.php

```diff
@@ -539,8 +539,9 @@ function initmorphy()
 // Moral/Physic attribute
 $morphys["phy"] = $langs->trans("Physical");
 $morphys["mor"] = $langs->trans("Moral");
+print '<tr class="morphy"><td class="titlefield">'.$langs->trans('MemberNature').' <span style="color: red">*</span></td><td>'."\n";
```

Comment:
Nearly. Because the ts is now open outside the if, you must also move the closing of tr in and td outside.

Reply:
Yes. The good answer was actually much simpler: there were 2 prints of the member nature when morphy was forced. With my last commit morphy is only hidden when forced :p